node/src/GlideClusterClient.ts

```diff
@@ -807,6 +810,47 @@ export class GlideClusterClient extends BaseClient {
         );
     }
 
+    /**
+     * Returns information about the functions and libraries.
+     *
+     * See https://valkey.io/commands/function-list/ for details.
+     *
+     * since Valkey version 7.0.0.
+     *
+     * @param options - Parameters to filter and request additional info.
+     * @param route - The command will be routed to a random route, unless `route` is provided, in which
+     *     case the client will route the command to the nodes defined by `route`.
```

Comment:
nit: this is a runaway sentence (also in Python).  Split into two sentences please.  
```
     * @param route - The client will route the command to the nodes defined by `route`. If not defined, the command will be routed to a random route.
```